rotkehlchen/chain/ethereum/airdrops.py

```diff
@@ -290,10 +317,10 @@ def get_airdrop_data(name: str, data_dir: Path) -> Iterator[list[str]]:
         # if not cached, get it from the gist
         try:
             response = requests.get(url=AIRDROPS[name][0], timeout=(30, 100))  # a large read timeout is necessary because the queried data is quite large  # noqa: E501
+            content = response.text
```

Comment:
why did you move this line?

Reply:
the `content` variable was possibly unbound in except block if we kept it in its try block
![image](https://github.com/rotki/rotki/assets/55619686/e9dbd25a-0684-4f83-a269-341433cea55f)